osu.Game.Tests/NonVisual/Filtering/FilterQueryParserTest.cs

```diff
@@ -246,6 +272,7 @@ public void TestApplyArtistQueriesWithSpaces()
             Assert.AreEqual("really like  yes", filterCriteria.SearchText.Trim());
             Assert.AreEqual(3, filterCriteria.SearchTerms.Length);
             Assert.AreEqual("name with space", filterCriteria.Artist.SearchTerm);
+            Assert.That(filterCriteria.Artist.Exact, Is.True);
```

Comment:
Please double check this during review. Currently this change forces all searches like `artist="this artist"` to exact mode. The quotes here were used to group the search to be scoped to artist, but not necessarily expecting exact match.

I think this seems fine, but worth double-checking with expectations.

Reply:
Probably fine until proven otherwise.